src/js/node/worker_threads.ts

```diff
@@ -77,11 +75,16 @@ function injectFakeEmitter(Class) {
   };
 
   function EventClass(eventName) {
-    if (eventName === "error" || eventName === "messageerror") {
-      return ErrorEvent;
-    }
+    switch (eventName) {
```

Comment:
is this change really necessary?

tbh this whole file is just style changes.

Reply:
As part of the bug fix, they are not necessary but harmless. They just make the coding style consistent in that file with the rest of the JS code in bun. 
They can be carved out that PR and be part of another code cleanup PR.